arithmetization/src/main/java/net/consensys/linea/sequencer/txselection/LineaTransactionSelectorCliOptions.java

```diff
@@ -20,40 +20,71 @@
 
 /** The Linea CLI options. */
 public class LineaTransactionSelectorCliOptions {
-  public static final int DEFAULT_MAX_BLOCK_CALLDATA_SIZE = 70000;
+  public static final int DEFAULT_MAX_BLOCK_CALLDATA_SIZE = 70_000;
   private static final String DEFAULT_MODULE_LIMIT_FILE_PATH = "moduleLimitFile.toml";
   public static final long DEFAULT_MAX_GAS_PER_BLOCK = 30_000_000L;
+  public static final int DEFAULT_VERIFICATION_GAS_COST = 1_200_000;
+  public static final int DEFAULT_VERIFICATION_CAPACITY = 90_000;
+  public static final int DEFAULT_GAS_PRICE_RATIO = 15;
+  public static final double DEFAULT_MIN_MARGIN = 0.9;
```

Comment:
```suggestion
  public static final double DEFAULT_MIN_MARGIN = 1.0;
```
I suggested 0.9 to be a default in our config, but 1.0 probably makes more sense as a sane default for the client code.